http/get_simple/java/server/pom.xml

```diff
@@ -0,0 +1,61 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!-- Licensed to the Apache Software Foundation (ASF) under one or more contributor
+  license agreements. See the NOTICE file distributed with this work for additional
+  information regarding copyright ownership. The ASF licenses this file to
+  You under the Apache License, Version 2.0 (the "License"); you may not use
+  this file except in compliance with the License. You may obtain a copy of
+  the License at http://www.apache.org/licenses/LICENSE-2.0 Unless required
+  by applicable law or agreed to in writing, software distributed under the
+  License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS
+  OF ANY KIND, either express or implied. See the License for the specific
+  language governing permissions and limitations under the License. -->
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <groupId>com.example</groupId>
+    <artifactId>ArrowHttpServer</artifactId>
+    <version>1.0-SNAPSHOT</version>
+
+    <properties>
+        <arrow.version>14.0.1</arrow.version>
```

Comment:
could use 15.0.0 and the bom?

Reply:
Do you have an example handy showing how to use the BOM?

Reply:
https://github.com/apache/arrow-adbc/blob/bf42dc64a98559b0d6602dd709dc18d05db9764f/java/pom.xml#L95-L104

https://github.com/apache/arrow-adbc/blob/bf42dc64a98559b0d6602dd709dc18d05db9764f/java/core/pom.xml#L25-L34

Reply:
Done. Let me know if I did this correctly. I guess it's unusual to have the `<dependencyManagement>` and `<dependencies>` in the same POM file, but I did that here given that it's a minimal example.

Reply:
LGTM